test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -36,11 +36,35 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, "a", "b", null);
     }
 
+    @Test
+    public void isAnyNull() throws Exception{
+        // all objects are null
+        assertIsNull(null, null);
+        assertIsNull(null, null, null);
+
+        // some objects are null
+        assertIsNull("abc", 123, null);
+        assertIsNull(null, 1, new Integer(1));
+
+        // no objects are null
+        assertIsNotNull("abc", "abc");
+        assertIsNotNull(1, new Integer(1));
+    }
+
+    private void assertIsNull(Object... objects) {
```

Comment:
For specific methods, you may try to follow the 3 part naming scheme mentioned in the tutorial. You can find more details on it in the coding standard.